src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -69,33 +69,34 @@ ucs_config_field_t uct_ib_mlx5_iface_config_table[] = {
 
     {"CQE_ZIPPING_ENABLE", "no",
      "Enable CQE zipping feature.\n"
-     "The CQE zipping reduces the PCI load by merging and compressing the almost identical CQEs to compression blocks.\n"
-     "In some cases this feature can have negative performance impact.",
+     "The CQE zipping reduces the PCI load by merging and compressing\n"
+     "almost identical CQEs to compression blocks.",
```

Comment:
"Enable CQE zipping feature. CQE zipping reduces PCI utilization by merging several similar CQEs\n"
"to a single CQE written by the device.",

Reply:
Shouldn't we use "device" without "the" article in this case?

Reply:
in this case we should use the